modules/mobile/docs/openapi.yaml

```diff
@@ -548,6 +548,179 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/facility/eligibility
+  /v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}/slots:
+    get:
+      description: \*EXPERIMENTAL* Accepts date range for a clinic at a va facility and returns available slots for a a direct schedule appointment.
+      operationId: slot
+      parameters:
+        - description: The start date for the range of appointments slots in ISO 8601 UTC
+            format. If not provided the start date will be considered today.
+          example: 2020-10-29T07:00:00Z
+          in: query
+          name: startDate
+          schema:
+            format: date-time
+            type: string
+        - description: The end date for the range of appointments slots in ISO 8601 UTC format.
+            If not provided the end date will be 2 months from today's date
+          example: 2021-11-29T08:00:00Z
+          in: query
+          name: endDate
+          schema:
+            format: date-time
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/ClinicSlots.yml
+          description: OK
+        '401':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Authentication is possible but has failed or not yet been provided.
+        '404':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: The requested resource could not be found.
+        '408':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: The response from this API timed out.
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Failure in processing entity
+        '500':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: An internal API error occured.
+        '503':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: An upstream service is unavailable, or its circuit breaker
+            may have been tripped.
+        '504':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: A response from an upstream service timed out.
+      security:
+        - Bearer: []
+      summary: /v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}/slots
+  /v0/appointments/facilities/{facililty_id}/clinics/{service_type}:
```

Comment:
Is this going to be equivalent to VAOS::V0::ClinicsController#index? If so, that controller also requires a `system_id` param. Do we need that?

Also, I think we should reconsider how this url is formed. If we ever needed to create a url like:
`/v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}`
It would collide with the url above. 

It might be better to either do it like:
`/v0/appointments/facilities/{facililty_id}/services/{service_type}/clinics`
or leave service type as a query param with a url like:
`/v0/appointments/facilities/{facililty_id}/clinics/`

I'm not sure which I prefer. Some might consider the first option to be bad form because service isn't a model nested under facilities. But at the same time, it's possible we could add a clinics index (like the second option) later that uses a different upstream service and doesn't require a service type (like the `VAOS::SystemsService#get_facility_clinics` does). 

Food for thought.

Reply:
that's a good point I do like `/v0/appointments/facilities/{facililty_id}/clinics/` with the query param more. 

the web currently uses the V2 ClinicsController which doesn't look to use that system_id so we should be good.